### PR TITLE
finishing touches on webpage

### PR DESCRIPTION
**Pull request by philippberens**
Hi Jeremy,

I added some small things to the webpage. Please review and merge the pull request.

### CODE REVIEW

File: client/components/panels/submit.js [hunk 1]
```diff
@@ -159,6 +163,11 @@ function submit (state) {
         <span style=${style.droptext}>drop result files here</span>
         ${status()}
       </div>
+	  <br>
+	  <div> <i>Disclaimer & Terms: By submitting you agree to cooperate with us in preparing an article describing the results of this challenge, by making a complete description of your algorithm and executable code in Matlab or Python available. Included in the article will be all algorithms beating the current state of the art (STM/oopsi) and their contributors will be co-authors on the paper. We will award the price money for winning the benchmark with respect to mean pearson correlation between true and inferred spike trains at 25 HZ computed on the test set. You data will not be shared with anyone without your consent, in particular not with our sponsor.
```
Comment (freeman-lab) [inline client/components/panels/submit.js]: I'd say "Matlab, Python, or other language of choice"

Comment (freeman-lab) [inline client/components/panels/submit.js]: "price" -> "prize"

**Comment by freeman-lab** [comment]
left a couple comments, otherwise looks good!

File: client/components/panels/submit.js (hunk 1 above)
Comment (philippberens) [reply client/components/panels/submit.js]: fixed

Comment (philippberens) [reply client/components/panels/submit.js]: fixd

**Comment by philippberens** [comment]
hi @freeman-lab,
your latest changes resulted in a conflict here, which I resolved. Maybe merge now?

P

**Comment by freeman-lab** [comment]
perfect, thanks!

**Pull request merged**